reader/components/toolbar/Toolbar.scss

```diff
@@ -45,33 +49,30 @@
 
       @include media-breakpoint-down(xs) {
         width: 35px;
-        padding: 0.30rem;
+        padding: 0.3rem;
       }
     }
   }
 
   .page-number-navigation {
-    margin-left: -1rem;
-    margin-right: -0.75rem;
-
     .input-change-page-number,
     .pages-count {
       display: inline-block;
-      width: 35px;
-      text-align: center;
+      width: 2.4rem;
     }
 
     .input-change-page-number {
+      text-align: right;
       background-color: inherit;
-      border: 0;
+      border: 1px solid transparent;
+      border-radius: 2px;
       box-shadow: none;
 
-      &:hover,
-      &:focus {
-        border: 1px solid $gray-300;
+      @include hover-focus {
+        border-color: $primary;
+        box-shadow: inset 0 1px 4px rgba(0, 0, 0, 0.1);
```

Comment:
Making input look more like an input. Creates tiny shadow to make the input look like it's under the surface.